okapi-core/src/test/java/org/folio/okapi/ModuleTest.java

```diff
@@ -2681,32 +2683,97 @@ private void undeployFirst(Handler<AsyncResult<Void>> fut) {
   }
 
   @Test
-  public void testInternalModule(TestContext context) {
-    logger.info("testInternalModule 1");
+  public void testInitdatabase(TestContext context) {
+    conf.remove("mongo_db_init");
+    conf.remove("postgres_db_init");
+    conf.put("mode", "initdatabase");
     async = context.async();
     undeployFirst(x -> {
-      conf.remove("mongo_db_init");
-      conf.remove("postgres_db_init");
+      DeploymentOptions opt = new DeploymentOptions().setConfig(conf);
+      vertx.deployVerticle(MainVerticle.class.getName(), opt, res -> {
+        context.assertTrue(res.succeeded());
+        async.complete();
+      });
+    });
+    async.await(1000);
+  }
```

Comment:
The duplicate code should be extracted into a method:

```
 private void undeployFirstAndDeploy(TestContext context, JsonObject aConf, Handler<AsyncResult<String>> fut) {
   Async myAsync = context.async();
   undeployFirst(context.asyncAssertSuccess(handler -> {
     DeploymentOptions opt = new DeploymentOptions().setConfig(aConf);
     vertx.deployVerticle(MainVerticle.class.getName(), opt, res -> {
       fut.handle(res);
       myAsync.complete();
     });
   }));
   myAsync.await(2000);
 }

 private void undeployFirstAndDeploy(TestContext context, JsonObject aConf) {
   undeployFirstAndDeploy(context, aConf, context.asyncAssertSuccess());
 }
```
Example usage:
```
undeployFirstAndDeploy(context, conf);
```
```
undeployFirstAndDeploy(context, conf, context.asyncAssertFailure(cause -> {
  Assert.assertThat(cause.getMessage(), containsString(
      "password authentication failed for user \"okapi\""));
}));
```

Reply:
ok. Done